docs/MEDIUM_ARTICLE_DRAFT.md

```
@@ -0,0 +1,251 @@
+    return () => clearTimeout(timer);
+  }
+}, []);
+```
+
+This helped with some edge cases, but the core issue remained.
+
+### The Real Culprit: EAS Secret Override
+
+After hours of debugging, we finally noticed something in the EAS build logs:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)